core/src/main/java/nl/nn/adapterframework/xml/SaxElementBuilder.java

```diff
@@ -78,7 +80,7 @@ public SaxElementBuilder addAttribute(String name, String value) throws SAXExcep
 		String attrlocalName = name;
 		String attrqName = attrlocalName;
 		String attrType = "";
-		attributes.addAttribute(attruri, attrlocalName, attrqName, attrType, value);
+		attributes.addAttribute(attruri, attrlocalName, attrqName, attrType, XmlUtils.normalizeAttributeValue(value));
```

Comment:
Kan jij voor de SaxElementBuilder en XmlBuilder nog een test toevoegen die de normalize in de addAttribute methode test?